packages/ui5-test-writer/test/index.test.ts

```diff
@@ -0,0 +1,108 @@
+import { generateOPAFiles, generatePageObjectFile } from '../src';
+import { join } from 'path';
+import { removeSync } from 'fs-extra';
+import type { Editor } from 'mem-fs-editor';
+
+describe('ui5-test-writer', () => {
+    let fs: Editor;
+    const debug = !!process.env['UX_DEBUG'];
+    const inputDir = join(__dirname, '/test-input');
+    const outputDir = join(__dirname, '/test-output');
+
+    beforeAll(() => {
+        removeSync(outputDir); // even for in memory
```

Comment:
Could you copy the `inputDir` contents to `outputDir` then use outputDir in the tests?
Currently running the following generates test files in the inputDir folder.
```
export UX_DEBUG=true 
pnpm run test
```

Reply:
ok